plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -82,6 +99,35 @@ func calculateResourceOccupancy(pod *api.Pod, podRequests *schedulercache.Resour
 	}
 }
 
+// Calculate the resource used on a node.  'node' has information about the resources on the node.
+// 'pods' is a list of pods currently scheduled on the node.
+// TODO: Use Node() from nodeInfo instead of passing it.
+func calculateUsedPriority(pod *api.Pod, podRequests *schedulercache.Resource, node *api.Node, nodeInfo *schedulercache.NodeInfo) schedulerapi.HostPriority {
```

Comment:
Personally, I think I would prefer having as is now (i.e. having an explicit function instead of using the other one). Then the behavior of those two are symetric.